department-of-veterans-affairs/caseflow
# Anya/4533 update vacols omo request

connects #4533 and #4527

To test:
1. Current user must be in the RequestStore
2. Run the following command from the rails console
```
decision_record = QueueRepository.find_case(vacols_id, attorney_css_id)
QueueRepository.reassign_case_to_judge(
    decision_record: decision_record, 
    decision_hash: 
            { work_product: "OMO - IME", 
              judge_css_id: "CF_MCCOY_317", 
              overtime: false, 
              document_id: "123456789.1234", 
              note: "Require action4" 
     } )
```

Testing results: 
1. Ran ```VACOLS::CaseAssignment.tasks_for_user("CASEFLOW_317")``` which returned 48 records
2. The ran ```decision_record = QueueRepository.find_case("3235908", "CASEFLOW_317")``` and ```QueueRepository.reassign_case_to_judge(decision_record: decision_record, decision_hash: { work_product: "OMO - IME",  judge_css_id: "CF_MCCOY_317", overtime: false, document_id: "123456789.1234", note: "Require action4" } )```
3. Then ran ```VACOLS::CaseAssignment.tasks_for_user("CASEFLOW_317")``` which returned 47 records
4. Checked the log output the ensure correct fields are updated.
